www/package.json

```diff
@@ -50,6 +53,7 @@
     },
     "devDependencies": {
         "@types/react-router-dom": "^5.1.7",
+        "@types/validator": "^13.1.4",
```

Comment:
Is there a reason this is in dev deps?

Reply:
No particular reason, I think that was the dev dependency flag was just set from wherever I copied from